plaidml2/bridge/keras/__init__.py

```diff
@@ -1060,7 +1060,7 @@ def not_equal(lhs, rhs):
 
 @_log_call
 def normalize_batch_in_training(x, gamma, beta, reduction_axes, epsilon=1e-3):
-    x_shape = shape(x)
+    x_shape = int_shape(x)
```

Comment:
This will cause problems with symbolic dimensions; I'm also not sure if this PR was the source of the problem you saw, I'll investigate more